.eslintrc.js

```diff
@@ -79,10 +79,11 @@ module.exports = {
       },
       plugins: ['@typescript-eslint/eslint-plugin'],
       rules: {
+        'react/jsx-uses-react': 'off',
+        'react/react-in-jsx-scope': 'off',
```

Comment:
turned this off since the [new jsx transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) released in react doesnt't needs react to be in scope